plugins/out_stackdriver/stackdriver.c

```diff
@@ -1862,6 +1862,7 @@ static void cb_stackdriver_flush(const void *data, size_t bytes,
     else {
         /* The request was issued successfully, validate the 'error' field */
         flb_plg_debug(ctx->ins, "HTTP Status=%i", c->resp.status);
+        flb_plg_debug(ctx->ins, "HTTP data=%s", c->resp.data);
```

Comment:
do not print the payload since it might be corrupted, you can get the data with _debug.http.response_payload configuration property

Reply:
Fixed by removing this line.